### PR TITLE
feat(apis): add interactions.js

### PR DESCRIPTION
**Pull request by mezotv**
This adds the popular interactions.js library used to create HTTP-only bots for discord.

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -1004,6 +1004,19 @@
     "tags": [],
     "urls": {}
   },
+  "interactions.js": {
```
Comment (Lulalaby) [inline libraries.json:1007]: that should be put one higher to follow the abc schema

File: languages.json [hunk 2]
```diff
@@ -115,7 +115,8 @@
       "discord.io",
       "discordie",
       "eris",
-      "js-cord"
+      "js-cord",
+      "interactions.js"
```
Comment (Lulalaby) [inline languages.json:119]:
```suggestion
      "interactions.js",
      "js-cord"
```

**Comment by mezotv** [comment]
I have gone ahead and fixed the issues should be correct now!

**Review by Lulalaby — APPROVED** [review]
Lgtm from my site.

@apacheli 👀

**Review by apacheli — APPROVED** [review]

**Pull request merged**